pyproject.toml

```diff
@@ -119,6 +119,7 @@ tests-only = [
   "pytest-mpl>=0.16",
   "pytest-xdist>=3.0.2",
   "pytest>=7.1.0",
+  "itables>=2.2.4",
```

Comment:
We will need to decide if this gets its own category and if we install it via "all" as well.